src/containers/profiles/ViewProfiles/ViewProfiles.scss

```diff
@@ -0,0 +1,5 @@
+.profile{
+  width: 150px !important;
```

Comment:
For a better image resolution try to use `max-width` and `max-height` or their `min` variation, they conserve the aspect ratio of the image

Reply:
Done